extension/src/testStatusBar.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { window, StatusBarAlignment, StatusBarItem } from "vscode";
+
+import * as Commands from "./commands";
+import { TestLevel, TestStatus, TestSuite } from "./protocols";
+
+export class TestStatusBar {
+    private statusBar: StatusBarItem = window.createStatusBarItem(StatusBarAlignment.Left, Number.MIN_VALUE);
+
+    public update(tests: TestSuite[]): void {
+        let failedCount: number = 0;
+        let passedCount: number = 0;
+        const testMethods: TestSuite[] = tests.map((t) => t.level === TestLevel.Method ? [t] : t.children)
+                                 .reduce((a, b) => a.concat(b));
+        for (const t of testMethods) {
+            if (t.level !== TestLevel.Method || !t.result) {
+                continue;
+            }
+            if (t.result.status === TestStatus.Fail) {
+                failedCount++;
+            } else if (t.result.status === TestStatus.Pass) {
+                passedCount++;
+            }
+        }
+        this.statusBar.text = '$(x)' + failedCount + '  $(check)' + passedCount;
+        this.statusBar.command = Commands.JAVA_TEST_SHOW_OUTPUT;

```

Comment:
Why this leads to the output instead of the corresponding test report?

Reply:
later on, we would support triggering tests from test explorer, we don't have a summary report for a trigger, while output and each trigger are 1:1 match, so i think directing to output panel is better